patientsearch/api.py

```diff
@@ -500,12 +577,18 @@ def external_search(resource_type):
         # See if local match already exists
         patient = resource_from_args(resource_type, request.args)
         try:
-            internal_bundle = internal_patient_search(token, patient)
+            internal_bundle = internal_patient_search(
+                token, patient, not reactivate_patient
```

Comment:
using keyword arguments will prevent getting them out of order, and make this easier to read/review.  i had to look up `internal_patient_search` to see what that third parameter is.  i.e. `active_only=not reactivate_patient`